src/modules/launcher/Plugins/Microsoft.Plugin.Program/Storage/Win32ProgramFileSystemWatchers.cs

```diff
@@ -28,8 +28,8 @@ private static string[] GetPathsToWatch()
         {
             string[] paths = new string[]
                             {
-                               Environment.GetFolderPath(Environment.SpecialFolder.Programs),
-                               Environment.GetFolderPath(Environment.SpecialFolder.CommonPrograms),
+                               Environment.GetFolderPath(Environment.SpecialFolder.StartMenu),
```

Comment:
@alekhyareddy28 Can you confirm where we got the list of special folders from?   Ideally this should be consistent with the start menu, and I'm wondering if we should be "adding" directories instead of replacing them.

Reply:
@ryanbodrug-microsoft This was inherited from wox: https://github.com/jjw24/Wox/blob/master/Plugins/Wox.Plugin.Program/Programs/Win32.cs#L348.

Reply:
@ryanbodrug-microsoft, Yes, we inherited these locations from Wox. While adding the file system watcher code, I just added the same locations to be watched as well for consistency.